joern-cli/frontends/swiftsrc2cpg/src/main/scala/io/joern/swiftsrc2cpg/astcreation/AstForDeclSyntaxCreator.scala

```diff
@@ -337,7 +337,7 @@ trait AstForDeclSyntaxCreator(implicit withSchemaValidation: ValidationMode) {
 
     val typeRefNode_ = typeRefNode(node, code(node), typeFullName)
     methodAstParentStack.find(_.isInstanceOf[NewMethod]).foreach { node =>
```

Comment:
```suggestion
    methodAstParentStack.collectAll[NewMethod].foreach { node =>
```

slightly different semantics (Iterable vs Option) though..